data/hai/hai culture conversations.txt

```diff
@@ -307,3 +307,24 @@ mission "Robotic Musical 1"
 				decline
 
 
+
+
+mission "Film Release"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		has "Hai Film Crew [3]: done"
+		random < 1
+	on offer
+		conversation
+			`Upon visiting the spaceport, you notice a poster for a movie in Hai. You recognize it as taking place on Darkrest, and quickly realize that this must be the movie that Likeet had made. The Hai-language title has been translated to "The Final Rest," but the full movie doesn't appear to have been subtitled in the human language yet.`
```

Comment:
```suggestion
			`Upon visiting the spaceport, you notice a poster for a movie in Hai. You recognize it as taking place on Darkrest, and quickly realize that this must be the movie that Likeet made. The Hai-language title has been translated to "The Final Rest," but the full movie doesn't appear to have been subtitled in the human language yet.`
```

Likeet hadn't already made it before meeting you, and they didn't commission it either.